### PR TITLE
[CSL-1734] Add a command to auxx for sending an update proposal

### PR DESCRIPTION
**Pull request by int-index**
* added a propose-unlock-stake-epoch command to auxx that sends an update proposal with a new value of unlockStakeEpoch
* added a print-bvd command auxx in order to monitor the current BlockVersionData
* fixed the name of puIdx (there was a TODO about it)

### CODE REVIEW

File: auxx/src/Command/Types.hs [hunk 1]
```diff
@@ -38,7 +39,7 @@ data SendToAllGenesisParams = SendToAllGenesisParams
 
 -- | Parameters for 'ProposeUpdate' command.
 data ProposeUpdateParams = ProposeUpdateParams
-    { puIdx             :: Int -- TODO: what is this? rename
+    { puSecretKeyIdx    :: Int -- the node that creates/signs the proposal
```
Comment (gromakovsky) [inline auxx/src/Command/Types.hs]: I wouldn't call it _the node_, it's the index of one of the secrets from our keyfile (`UserSecret`).

File: auxx/src/Command/Update.hs [hunk 2]
```diff
@@ -126,13 +131,65 @@ propose sendActions ProposeUpdateParams{..} = do
...
+            , bvmMaxTxSize         = Nothing
+            , bvmMaxProposalSize   = Nothing
+            , bvmMpcThd            = Nothing
+            , bvmHeavyDelThd       = Nothing
+            , bvmUpdateVoteThd     = Nothing
+            , bvmUpdateProposalThd = Nothing
+            , bvmUpdateImplicit    = Nothing
+            , bvmSoftforkRule      = Nothing
+            , bvmTxFeePolicy       = Nothing
+            , bvmUnlockStakeEpoch  = Just puseUnlockStakeEpoch
```
Comment (gromakovsky) [inline auxx/src/Command/Update.hs]: I would define a `BlockVersionModifier` with all fields set to `Nothing` somewhere and use it here (and probably in the command above).

Comment (int-index) [reply auxx/src/Command/Update.hs]: As a matter of fact, I recall that it was defined somewhere. Good idea to use it.

**Comment by gromakovsky** [comment]
Please don't restart builds, there have failing tests.

**Comment by gromakovsky** [comment]
I rebased on the latest master to fix CI.

**Comment by gromakovsky** [comment]
TODO: we will need to update documentation about proposing an update after it's merged to this branch.

**Pull request merged**